google/cloud/datastore_v1/gapic/datastore_client.py

```diff
@@ -464,9 +547,9 @@ def begin_transaction(
     def commit(
         self,
         project_id,
-        mode,
-        mutations,
+        mode=None,
         transaction=None,
+        mutations=None,
```

Comment:
This is breaking because the arg order changes. We should patch this with a synth replace temporarily